docs/program/extend/modular-resources/examples/add-tflite-module.md

```diff
@@ -0,0 +1,325 @@
+   - [`CMAKE_INSTALL_PREFIX`](https://github.com/viamrobotics/viam-cpp-sdk/blob/main/BUILDING.md#cmake_install_prefix) to install to <file>~/example_workspace/opt</file> instead of the default <file>./install</file> location.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Download the `yamnet/classification` model file
+
+This example uses the `yamnet/classification` TensorFlow Lite model for audio classification.
+
+1. Download the `yamnet/classification` TensorFlow Lite model file and place it in your <file>example_workspace</file> directory:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   curl -Lo ~/example_workspace/lite-model_yamnet_classification_tflite_1.tflite https://tfhub.dev/google/lite-model/yamnet/classification/tflite/1?lite-format=tflite
+   ```
+
+   Alternatively, you may download the model file here: [yamnet classification tflite model](https://tfhub.dev/google/lite-model/yamnet/classification/tflite/1).
+   If you download in this fashion, move the downloaded file to your <file>~/example_workspace</file> directory.
+
+1. Extract the labels file <file>yamnet_label_list.txt</file> from the downloaded model file:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   unzip ~/example_workspace/lite-model_yamnet_classification_tflite_1.tflite
+   ```
+
+   The labels file provides pre-populated labels for the calculated scores, so that output scores can be associated and returned with their matching labels.
+   You can omit this file if desired, which will cause the inference client to return the computed scores without labels.
+
+## Install `viam-server`
+
+Next, install `viam-server` on your robot, if you have not done so already:
+
+1. Navigate to [the Viam app](https://app.viam.com) in your browser and [add a new robot](/manage/fleet/robots/#add-a-new-robot).
+
+1. Switch to the **Setup** tab, and select your platform from the **Mode** selection at the top.
+   If you are installing on a Linux system, additionally select your system's **Architecture**.
+
+1. Follow the steps listed under the **Setup** tab to install `viam-server` on your system.
+
+   {{< alert title="Important" color="note" >}}
+   If you are installing `viam-server` within the `bullseye` Docker container provided with the C++ SDK, you will need to run the following command *instead* of the command listed in step 2 **Download and install viam-server** in the Viam app:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   curl https://storage.googleapis.com/packages.viam.com/apps/viam-server/viam-server-stable-x86_64.AppImage -o viam-server && chmod 755 viam-server && sudo ./viam-server --appimage-extract-and-run -config /etc/viam.json
+   ```
+
+   Once installed within the Docker container, you can later run `viam-server` with the following command, from within the directory you installed it to:
```

Comment:
```suggestion
  The command above starts `viam-server`.
   If at some point you stop `viam-server`, you can restart it with the following command, from within the directory you installed it to:
```
[opt] I kinda think "you can later run..." is a bit ambiguous. Folks might wonder when "later" is and why they'd need to run it, and whether they need to run it now.